_episodes/04-data-types-and-format.md

```diff
@@ -15,7 +15,11 @@ objectives:
     - "Analyze datasets having missing/null values (NaN values)."
     - "Write manipulated data to a file."
 keypoints:
-    - "FIXME"
+    - "Pandas and base Python use slightly different names for data types"
```

Comment:
Just to make my previous comment clearer:
1. Let's not use "base Python", just Python. I suggest we don't introduce new terms in this lesson so that learners don't remember terms that do not exist in the wild. Hence, I suggest removing the word "base" from the episode (in a different PR).
2. One can't say that "Pandas and Python use different names for data types" because Pandas and Python are different beasts. It's Pandas that uses different (from Python's) names for data types, for example: "object" for textual data. So, you can say something like:
```
Pandas uses different (from Python's) names for data types, for example: "object" for textual data.
```

Reply:
1. I will recommend reading episode 04 line 62 https://github.com/datacarpentry/python-ecology-lesson/blob/gh-pages/_episodes/04-data-types-and-format.md, the wording base Python has been already introduced.

2. Keypoints are related to the episode and come at the end, hence there are no examples, they have seen the examples and the keypoints are only a summary.

what about this?

"The library Pandas uses other names for data types than base Python"

Reply:
Remove unnecessary words:

> Pandas uses other names for data types than Python.

Reply:
Regarding the "example" (`"object" for textual data.`) and keypoints in general: 1. I know where keypoints appear in the episode. 2. it repeats the material that some of the participants have been taught for the first time, so repeating it once again is not a bad thing.

Reply:
I'm mostly ambivalent on the use of Python vs. base Python. I sort of lean towards "base Python", since learners have a hard time with what is Python vs. what is the shell vs. what is Pandas. But I really like the idea of including an example - particularly the object type, since it comes up a lot, and will come up in future lessons.